README.md

```diff
@@ -58,6 +59,15 @@ datagalaxy-toolbox.exe copy-attributes [-h] --url-source URL_SOURCE --url-target
 datagalaxy-toolbox.exe copy-technologies [-h] --url-source URL_SOURCE --url-target URL_TARGET --token-source TOKEN_SOURCE --token-target TOKEN_TARGET
 ```
 
+#### copy-screens
+
+```
+datagalaxy-toolbox.exe copy-screens [-h] --url-source URL_SOURCE [--url-target URL_TARGET] --token-source TOKEN_SOURCE [--token-target TOKEN_TARGET] [--workspace-source WORKSPACE_SOURCE] [--workspace-target WORKSPACE_TARGET]
```

Comment:
It might be more explicit to introduce two commands:

```
datagalaxy-toolbox.exe copy-workspace-screens
```


```
datagalaxy-toolbox.exe copy-client-screens
```

Reply:
Actually the command can have 4 different scenarios : 
1. Copy screens from a clientspace to a clientspace
2. Copy screens from a clientspace to a workspace
3. Copy screens from a workspace to a workspace
4. Copy screens from a workspace to a clientspace

I will try to make it more clear, either with a longer explanation, or with two commands as you suggested

Reply:
I didn't know we were using the exact same format between clientspace screens and workspace screens 🤔

Reply:
Yes it is the same and it is very practical here